internal/backends/sqlite/backend_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlite
+
+import (
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewBackend(t *testing.T) {
+	t.Parallel()
+
+	err := os.MkdirAll("tmp/dir", os.ModePerm)
```

Comment:
Is the reason not using `t.TempDir()` because we want to use current dir `.` notation? :thinking:

Reply:
Yes, exactly 😀